res/css/views/elements/_SSOButtons.scss

```diff
@@ -46,13 +46,13 @@ limitations under the License.
     }
 
     .mx_SSOButton_default {
-        color: $button-primary-bg-color;
-        background-color: $button-secondary-bg-color;
-        border-color: $button-primary-bg-color;
+        color: $button-secondary-bg-color;
+        background-color: $button-primary-bg-color;
+        border-color: $button-secondary-bg-color;
     }
     .mx_SSOButton_default.mx_SSOButton_primary {
-        color: $button-primary-fg-color;
-        background-color: $button-primary-bg-color;
+        color: $button-primary-bg-color;
```

Comment:
This change looks odd as the foreground and background colours have been swapped?

Reply:
it's all good

Reply:
this change needed for light color button now instead of white button previously for SSO

Reply:
done